src/addresstype.cpp

```diff
@@ -87,10 +87,10 @@ bool ExtractDestination(const CScript& scriptPubKey, CTxDestination& addressRet)
         addressRet = tap;
         return true;
     }
-    case TxoutType::WITNESS_UNKNOWN: {
+    case TxoutType::WITNESS_UNKNOWN:
+    case TxoutType::ANCHOR:
         addressRet = WitnessUnknown{vSolutions[0][0], vSolutions[1]};
         return true;
-    }
     case TxoutType::MULTISIG:
     case TxoutType::NULL_DATA:
     case TxoutType::NONSTANDARD:
```

Comment:
In 7013da222720fde5ea1c76bba99b1c91ae18558f "policy: Add OP_TRUE <0x4e73> as a standard output type"

Add `ANCHOR` txout script type description in the `CTxDestination` docstring.

Reply:
I'm actually having trouble figuring out where this stuff is exposed since we wouldn't be issuing these addresses as a wallet. Mind if I punt on this?

Reply:
ended up adding it since its required if we aren't returning the "solutions" vector